Tasks/src/Common_Array.java

```diff
@@ -0,0 +1,40 @@
+import java.util.Arrays;
+
+public class Common_Array
+{
+    public static void main(String[] args)
+    {
+        int [] arr1={1,5,8,4,6,3};
+        int [] arr2={1,5,8,4,5,3};
+        int [] new_array=common_members(arr1,arr2);
+
+        for (int i =0 ;i < new_array.length;i++)
+        {
+
+            System.out.print(new_array[i] + ",");
+        }
+
+    }
+    public  static int[] common_members(int []arr1 , int []arr2 )
```

Comment:
method names should be written as camel case and start with a verb. ex, combineCommonValues